Xwt.XamMac/Xwt.Mac/FileDialogBackend.cs

```diff
@@ -64,6 +65,12 @@ protected virtual void OnInitialize (System.Collections.Generic.IEnumerable<File
 		public bool Run (IWindowFrameBackend parent)
 		{
 			var returnValue = Panel.RunModal ();
+			if (parent != null) {
+				var win = parent as NSWindow ?? Runtime.GetNSObject (parent.NativeHandle) as NSWindow;
```

Comment:
can we have an internal extension method on IWindowFrameBackend for this?

Reply:
In fact, we should not even have separate implementations in Xwt.Gtk.Mac